consensus/polybft/consensus_runtime.go

```diff
@@ -592,7 +617,7 @@ func (c *consensusRuntime) startEventTracker() error {
 // deliverMessage receives the message vote from transport and inserts it in state db for given epoch.
 // It returns indicator whether message is processed successfully and error object if any.
 func (c *consensusRuntime) deliverMessage(msg *TransportMessage) (bool, error) {
-	_, epoch := c.getLastBultBlockAndEpoch()
+	_, epoch := c.getLastBuiltBlockAndEpoch()
```

Comment:
You can use getEpoch function here

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/865/commits/1d887e7738aa86a285d3ad490808352f95389448